scripts/bash/run_enceladus.sh

```diff
@@ -381,37 +385,41 @@ if [ "$DRA_ENABLED" = true ] ; then
     add_spark_conf_cmd "spark.dynamicAllocation.enabled" "true"
     add_spark_conf_cmd "spark.shuffle.service.enabled" "true"
     add_spark_conf_cmd "spark.sql.adaptive.enabled" "true"
-    add_spark_conf_cmd "spark.dynamicAllocation.maxExecutors" ${DRA_MAX_EXECUTORS}
-    if [ ! -z "$DRA_MIN_EXECUTORS" ]; then
-        add_spark_conf_cmd "spark.dynamicAllocation.minExecutors" ${DRA_MIN_EXECUTORS}
+    add_spark_conf_cmd "spark.dynamicAllocation.maxExecutors" "${DRA_MAX_EXECUTORS}"
+    if [ -n "$DRA_MIN_EXECUTORS" ]; then
+        add_spark_conf_cmd "spark.dynamicAllocation.minExecutors" "${DRA_MIN_EXECUTORS}"
     fi
-    if [ ! -z "$DRA_ALLOCATION_RATIO" ]; then
-        add_spark_conf_cmd "spark.dynamicAllocation.executorAllocationRatio" ${DRA_ALLOCATION_RATIO}
+    if [ -n "$DRA_ALLOCATION_RATIO" ]; then
+        add_spark_conf_cmd "spark.dynamicAllocation.executorAllocationRatio" "${DRA_ALLOCATION_RATIO}"
     fi
-    if [ ! -z "$ADAPTIVE_TARGET_POSTSHUFFLE_INPUT_SIZE" ]; then
-        add_spark_conf_cmd "spark.sql.adaptive.shuffle.targetPostShuffleInputSize" ${ADAPTIVE_TARGET_POSTSHUFFLE_INPUT_SIZE}
+    if [ -n "$ADAPTIVE_TARGET_POSTSHUFFLE_INPUT_SIZE" ]; then
+        add_spark_conf_cmd "spark.sql.adaptive.shuffle.targetPostShuffleInputSize" "${ADAPTIVE_TARGET_POSTSHUFFLE_INPUT_SIZE}"
     fi
 fi
 
 JVM_CONF="spark.driver.extraJavaOptions=-Dstandardized.hdfs.path=$STD_HDFS_PATH \
 -Dspline.mongodb.url=$SPLINE_MONGODB_URL -Dspline.mongodb.name=$SPLINE_MONGODB_NAME -Dhdp.version=$HDP_VERSION \
 $MT_PATTERN"
 
+if [ "$HELP_CALL" == "1" ]; then
+  source ${SRC_DIR}/help_script.sh
+fi
```

Comment:
Won't the script continue after displaying the help?

Reply:
I tested it and it used the exit from the `help_script.sh`

Reply:
Yeah, but if 0. Won't it continue to execute the `run_enceladus.sh`?

Reply:
Or the `exit` is global, not just from the current level?

Reply:
But even if, not sure it's good to depend on the `exit` somewhere else. 🤔

Reply:
> But even if, not sure it's good to depend on the `exit` somewhere else. 🤔

I am thinking about it and you are right. Will move the exit here.